ElectronNET.CLI/ElectronNET.CLI.csproj

```diff
@@ -33,7 +33,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <WarningLevel>4</WarningLevel>
```

Comment:
was this intentional?

Reply:
No, it was an automatic change by Visual Studio.
I reverted it in the latest commit.